v2.1/configure-replication-zones.md

```diff
@@ -52,24 +52,14 @@ When replicating data, whether table or system, CockroachDB always uses the most
 
 ## Replication zone format
 
-A replication zone is specified in [YAML](https://en.wikipedia.org/wiki/YAML) format and looks like this:
+Use the [`CONFIGURE ZONE`](configure-zone.html) [statement](sql-statements.html) to set a replication zone:
 
-~~~ yaml
-range_min_bytes: <size-in-bytes>
-range_max_bytes: <size-in-bytes>
-gc:
-  ttlseconds: <time-in-seconds>
-num_replicas: <number-of-replicas>
-constraints: <json-formatted-constraints>
+{% include copy-clipboard.html %}
+~~~ sql
+> ALTER TABLE t CONFIGURE ZONE USING range_min_bytes = 0, range_max_bytes = 90000, gc.ttlseconds = 89999, num_replicas = 4, constraints = '[-region=west]';
 ~~~
 
-Field | Description
-------|------------
-`range_min_bytes` | Not yet implemented.
-`range_max_bytes` | The maximum size, in bytes, for a range of data in the zone. When a range reaches this size, CockroachDB will spit it into two ranges.<br><br>**Default:** `67108864` (64MiB)
-`ttlseconds` | The number of seconds overwritten values will be retained before garbage collection. Smaller values can save disk space if values are frequently overwritten; larger values increase the range allowed for `AS OF SYSTEM TIME` queries, also know as [Time Travel Queries](select-clause.html#select-historical-data-time-travel).<br><br>It is not recommended to set this below `600` (10 minutes); doing so will cause problems for long-running queries. Also, since all versions of a row are stored in a single range that never splits, it is not recommended to set this so high that all the changes to a row in that time period could add up to more than 64MiB; such oversized ranges could contribute to the server running out of memory or other problems.<br><br>**Default:** `90000` (25 hours)
-`num_replicas` | The number of replicas in the zone.<br><br>**Default:** `3`
-`constraints` | A JSON object or array of required and/or prohibited constraints influencing the location of replicas. See [Types of Constraints](#types-of-constraints) and [Scope of Constraints](#scope-of-constraints) for more details.<br/><br/>To prevent hard-to-detect typos, constraints placed on [store attributes and node localities](#descriptive-attributes-assigned-to-nodes) must match the values passed to at least one node in the cluster. If not, an error is signalled.<br/><br/>**Default:** No constraints, with CockroachDB locating each replica on a unique node and attempting to spread replicas evenly across localities.
+{% include v2.1/zone-configs/variables.md %}
```

Comment:
We already discussed updating `lease_preferences` - please feel free to steal from my PR in #3850 as needed